.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
Greatjob including 
.gitignore
eslintrc
babelrc 
package.json
and travis.yml